orion/sites/admin/src/location/locationEdit.js

```diff
@@ -88,71 +74,41 @@ export class LocationEditPage extends React.Component {
                 .then((res) => successCallback())
                 .catch((err) => failCallback(err));
         }
-    }
+    };
 
-    onDeleted() {
+    onDeleted = () => {
         const locId = this.props.locId;
         API.delete("api/locations/v1/location/" + locId).then((res) => {
             window.location.hash = "locations";
         });
-    }
+    };
 
-    onSaved() {
+    onSaved = () => {
```

Comment:
do this arrow thingy! It basically means a function that takes in no parameters does this ` => {...}`